backend/notifications/Models/EmailResponse.cs

```diff
@@ -10,16 +10,31 @@ namespace Pims.Notifications.Models
     public class EmailResponse
     {
         #region Properties
```

Comment:
After I added comments to this file coverage is complaining I don't have coverage for these comments... Which is idiotic...

Reply:
Guess they really want to check our grammar 😛